client/settings/customization-option-notice/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { __ } from '@wordpress/i18n';
+import interpolateComponents from 'interpolate-components';
+import { Icon, closeSmall } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import UpeContext from '../upe-toggle/context';
+
+const NoticeWrapper = styled.div`
```

Comment:
A `Notice` component already exists in `@wordpress/components`, it just needs some adjustments on the margins and the background color. Storybook: https://wordpress.github.io/gutenberg/?path=/story/components-notice--default - WCPay implementation: https://github.com/Automattic/woocommerce-payments/blob/develop/client/components/inline-notice/ . It also includes a `isDismissible` prop.

What do you think about reusing that one?

Reply:
Thanks for this. I did not know this exists but as it does reusing is the best approach.